Training/PythonWebServer/webserver.py

```diff
@@ -2,16 +2,34 @@
 
 from http.server import SimpleHTTPRequestHandler, HTTPServer
 import sys
+import psutil
+from datetime import datetime
+import pytz
+import json
```

Comment:
In general, imports are best in the order

- system
- third party
- internal

With a space separating each section.